content/backend/graphql-python/1-getting-started.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Getting Started
+description: Setting up a Django Graphene Project
+---
+
+### What are you going to build?
+In this tutorial, you are going to create a [Hackernews](https://news.ycombinator.com) clone using Django and Graphene. The platform will have the following features:
+
+* List of Users and Links
+* Users creation and authentication
+* Users can create Links and vote on them
+
+### Creating your local environment
+Before starting, make sure you have Python 3.6 installed. If not, you can download it [here](https://www.python.org/downloads/). The installation is pretty straightforward on any supported operating system.
+
+When working with Python, you will use *Virtual Environment*: a tool for keeping all the project's dependencies isolated from your system – or other projects.
+
+<Instruction>
```

Comment:
This is not actually rendered as an instruction block on the site because the code block isn't the last element in the `<Instruction>` tags:

![image](https://user-images.githubusercontent.com/4058327/29417361-41a89032-8369-11e7-9bc4-6b16ac650ab7.png)

Currently our Markdown parser requires that, if an instruction block contains a code block, then the code block _must_ be the last element in the instruction block (so it can't be followed by a paragraph as is the case here).